contracts/Market.sol

```diff
@@ -927,7 +936,12 @@ contract Market is
         amount: unrestrictedSupplierFee
       });
     }
-    bytes memory data = abi.encode(recipient, certificateAmount);
+    bytes memory data = abi.encode(
+      recipient,
+      certificateAmount,
+      address(_purchasingToken),
+      _priceMultiple
+    );
```

Comment:
What's this slither alert 👇?

Reply:
ah yeah another divide before multiply... I think I can take care of that